components/elements/frontPage/LoginForm.js

```diff
@@ -25,19 +25,26 @@ const LoginForm = ({ className, logo }) => {
       (async () => {
         try {
           dispatch({ type: INIT });
-          const response = await slsFetch("/api/login", {
+          const response = await slsFetch("/api/auth/login", {
```

Comment:
I agree with you, this needs to be refactored.

However, can we do this in a more `REST-ful` manner?

Logging in and out can be viewed as creating and deleting a session.

Login - `POST /api/session`
Logout - `DELETET /api/session`

Reply:
noted!